src/base-element.js

```diff
@@ -334,6 +334,16 @@ export class BaseElement {
     // Subclasses may override.
   }
 
+  /**
+   * Override in subclass to adjust the element when it is being added to
+   * the DOM. Could e.g. be used to add a listener. Notice, that this
+   * callback is called immediately after `buildCallback()` if the element
+   * is attached to the DOM.
+   */
+  attachedCallback() {
```

Comment:
This is great. @dmanek and I were just discussing in #30901 about where event listeners should be. It's really not consistent between components.

Reply:
Yes. We definitely have an issue now. Hopefully all Bento components will only have external listeners running from within the Preact component itself. In that case it'd be handled fully automatically for us.